### PR TITLE
Project Business site

### PR DESCRIPTION
**Pull request by EmmaaBerg**

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,253 @@
...
+   color: #FFFFFF;
+   background-color: #884B58;
+   padding: 0.6rem;
+   font-size: 15px;
+   text-align:left;
+   border: 2px solid #6D2E46;
+ }
+
+ .accordion .accordion-element .question::before{
+   content:"+";
```
Comment (joannaringqvist) [inline code/style.css:176]: 🥳 This was good!

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,253 @@
...
+  border-radius: 5px;
+}
+
+input[type=radio] {
+  margin-top: 10px;
+}
 
-.active + .answer {
-  /* some styling here */
+input:focus {
```
Comment (joannaringqvist) [inline code/style.css:104]: This was good for me to know. I will try to remember it for the future. 👍

**Review by joannaringqvist — APPROVED** [review]
Great code! And also good that you had comments for every section in the CSS.

File: code/index.html [hunk 3]
```diff
@@ -1,30 +1,105 @@
...
+              <input id="email" type="email" name="email" placeholder="Enter your email" required />
+            </label>
+
+            <label>
+              Password
+              <input id="password" type=password placeholder="Create a password" required />
+            </label>
+          </div>
+
+          <label>
```
[the remark this answers is not in the document]
Comment (EmmaaBerg) [reply code/index.html:46]: Ah thank you, I will look at that! :)

**Pull request closed without merging**